docs/custom-geospatial-query-extension.md

```diff
@@ -1,124 +1,119 @@
+  -H "Content-Type: application/json; charset=UTF-8" \
+  -d "{\"pairsPayload\":\"<QUERY_PAYLOAD>\",\"analyticsName\":\"<ANALYTICS_NAME\"}"
+```
 
 <b>Example Response</b>:
 
-`
+``` json
 [
-  {
-    "analyticsUuid": "1f2d5a9e-39c5-4ca4-bebc-14ac43646960",
-    "layerId": "51",
-    "baseComputationId": "1607533200_04490762"
-  }
-]`
-
-
-
+  {
+    "analyticsUuid": "1f2d5a9e-39c5-4ca4-bebc-14ac43646960",
+    "layerId": "51",
+    "baseComputationId": "1607533200_04490762"
+  }
+]
+```
 
 ### Registration Part # 2 - Visualization metadata
 
 
- Before making the sample request below, modify the <b>`id`</b> to be something unique, set the desired <b>`displayName`</b> and substitute the correct <b>`data Attribute uuid`</b> using the `analyticsUuid` value from the response above. Also, substitute JWT_TOKEN
- 
- Everything else can be kept as is for an initial simplified scenario. Contact your IBM representative to discuss advanced geospatial analytics use-cases. 
+ Before making the sample request below, modify the <b>`id`</b> to be something unique, set the desired <b>`displayName`</b> and substitute the correct <b>`data Attribute uuid`</b> using the `analyticsUuid` value from the response above. Also, substitute ACCESS_JWT
 
+ Everything else can be kept as is for an initial simplified scenario. Contact your IBM representative to discuss advanced geospatial analytics use-cases.
 
-`curl --location --request PUT 'https://api.wsitrader.com/api/v1/IMAP/put-layer-config-block' \
---header 'Authorization: Bearer <JWT_TOKEN>' \
+
+```
+curl --location --request PUT 'https://api.wsitrader.com/api/v1/IMAP/put-layer-config-block' \
+--header 'Authorization: Bearer <ACCESS_JWT>' \
 --header 'Content-Type: application/json' \
 --data-raw '{VIEWERSHIP_ROLE : 'ALL', CONFIG_BLOCK: {
-    <b>id: 'customQuery-staging-test01'</b>,
-    modelRegistryId: null,
-    <b>displayName: 'Staging Test - Custom Query 01'</b>,
-    provider: null,
-    layerType: 'grid',
-    isSelected: false,
-    isActive: false,
-    enableValidity: false,
-    lastUpdatedUtc: null,
-    coverageArea: 'Custom',
-    dataAttributes: {
-      url: 'https://foundation.agtech.ibm.com/v2',
-      <b>uuid: '1f2d5a9e-39c5-4ca4-bebc-14ac43646960'</b>
-    },
-    menuIconUrl: null,
-    legendUrl: '',
-    styleProperties: {
-      palette: {
-        COLOR_STEPS: [
-          { step: -1, rgba: [ 0, 0, 8, 255 ] },
-          { step: 0, rgba: [ 11, 0, 251, 255 ] },
-          { step: .2, rgba: [ 236, 0, 34, 255 ] },
-          { step: .4, rgba: [ 250, 93, 7, 255 ] },
-          { step: .6, rgba: [ 250, 249, 0, 255 ] },
-          { step: .8, rgba: [ 0, 239, 0, 255 ] },
-          { step: 1, rgba: [ 1, 49, 1, 255 ] }
-        ]
-      },
-      unit: 'C',
-      isInterpolated: true,
-      extendMinimumColor: false,
-      extendMaximumColor: true,
-      invalidDataValue: -9999
-    }
-  }}'`
-
-
-<b>Example response</b>: 
+    id: 'customQuery-staging-test01',
```

Comment:
This `PUT` data payload needs further refinement from the look of the JSON and quoting for usage in shell commands.

Reply:
Updated the JSON payload syntax to at least pass when provided to `jq -r .`